Jenkinsfile

```diff
@@ -26,10 +26,18 @@ node("master") {
 */
 
 def helpers = load 'build-helpers.groovy'
+def submitResultList = []
 
 import org.codehaus.groovy.runtime.*;
 
 CQZ_BUILD_ID = DateGroovyMethods.format(new Date(), 'yyyyMMddHHmmss')
+CQZ_VERSION = ""
+
+BUILD_STATUS_FAILED = 'Failed'
+BUILD_STATUS_SUCCESSFUL = 'Successful'
+
+EMAIL_LIST = 'alexander@cliqz.com'
```

Comment:
would be nice to set those as job parameters, so we wont receive accidental emails if someone else is trying to build ;-)